### DISCUSSION TITLE
Board Sync

### DISCUSSION THREAD

**Discussion by w3333** [yzyray/lizzieyzy#203] [Ideas]
Hi,

so firstly, thanks for continuing development on Lizzie.

However: the Board Synchronization feature seems to be specifically made for online cheating. Why else would you need to sync to an active game board from Tygem, Fox or Sina? So please reconsider this feature. It only enables people to cheat even more than they already do. Do you really want to destroy online Go? It is a plague already, please stop spreading it.

**Comment by ernestasromeika** [comment]
I think it's useful when reviewing an ongoing pro game/broadcast game. And removing this feature won't stop people from cheating.
